cmds/exp/pox/pox.go

```diff
@@ -139,30 +220,70 @@ func pox() error {
 		defer os.RemoveAll(dir)
 	}
 
-	lo, err := loop.New(*file, "squashfs", "")
-	if err != nil {
-		return err
-	}
-	defer lo.Free() //nolint:errcheck
+	if *zip {
+		c := exec.Command("unzip", *file, "-d", dir)
+		if err = c.Run(); err != nil {
+			log.Printf("Unzipping: %v", err)
+		}
+	} else {
+		lo, err := loop.New(*file, "squashfs", "")
+		if err != nil {
+			return err
+		}
+		defer lo.Free() //nolint:errcheck
 
-	mountPoint, err := lo.Mount(dir, 0)
-	if err != nil {
-		return err
+		mountPoint, err := lo.Mount(dir, 0)
+		if err != nil {
+			return err
+		}
+		defer mountPoint.Unmount(0) //nolint:errcheck
+	}
+	for _, m := range chrootMounts {
+		err := syscall.Mount(m.source, dir+m.target, m.fstype,
```

Comment:
Use pkg/mount.Mount

It returns a mountpoint, error, and you can then defer mp.Unmount(0) just like the loop mount.

Reply:
Use filepath.Join instead of +